content/docs/intro/rbac.md

```diff
@@ -0,0 +1,147 @@
+
+Additionally, you may also create a RoleBinding with `cluster-admin` access. Granting a user `cluster-admin` access at the namespace scope provides full control over every resource in the namespace, including the namespace itself.
+
+For this example, we will create a user with the `edit` Role. First, create the namespace:
+
+```
+$ kubectl create namespace foo
+```
+
+Now, create a RoleBinding in that namespace, granting the user the `edit` role.
+
+```
+$ kubectl create rolebinding sam-edit
+    --clusterrole edit \​
+    --user sam \​
+    --namespace foo
+```
+
+### Example: Grant a user read/write access at the cluster scope
+
+If a user wishes to install a chart that installs cluster-scope resources (namespaces, roles, custom resource definitions, etc.), they will require cluster-scope write access.
+
+To do that, grant the user either `admin` or `cluster-admin` access.
+
+Granting a user `cluster-admin` access grants them access to absolutely every resource available in Kubernetes, including node access with `kubectl drain` and other administrative tasks. It is highly recommended to consider providing the user `admin` access instead, or to create a custom ClusterRole tailored to their needs.
+
+```
+$ kubectl create clusterrolebinding sam-view
+    --clusterrole view \​
+    --user sam
+
+$ kubectl create clusterrolebinding sam-secret-reader
+    --clusterrole secret-reader \​
+    --user sam
+```
+
+### Example: Grant a user read-only access to a particular namespace
+
+You might've noticed that there is no ClusterRole available for viewing secrets. The `view` ClusterRole does not grant a user read access to Secrets due to escalation concerns. Helm stores release metadata as Secrets by default.
+
+In order for a user to run `helm list`, they need to be able to read these secrets. For that, we will create a special `secret-reader` ClusterRole.
+
+Create the file `cluster-role-secret-reader.yaml` and write the following content into the file:
+
+```
+apiVersion: rbac.authorization.k8s.io/v1​
+kind: ClusterRole​
+metadata:​
+  name: secret-reader​
+rules:​
+- apiGroups: [""]​
+  resources: ["secrets"]​
+  verbs: ["get", "watch", "list"]
+```
+
+Then, create the ClusterRole using
+
+```
+$ kubectl create -f clusterrole-secret-reader.yaml​
+```
+
+Once that's done, we can grant a user read access to most resources, and then grant them read access to secrets:
+
+```
+$ kubectl create namespace foo
+
+$ kubectl create rolebinding sam-view
+    --clusterrole view \​
+    --user sam \​
+    --namespace foo
+
+$ kubectl create rolebinding sam-secret-reader
+    --clusterrole secret-reader \​
+    --user sam \​
+    --namespace foo
+```
+
+### Example: Grant a user read-only access at the cluster scope
+
+In certain scenarios, it may be beneficial to grant a user cluster-scope access. For example, if a user wants to run the command `helm list --all-namespaces`, the API requires the user has cluster-scope read access.
```

Comment:
@bacongobbler - Am I the only one that thinks this is insane?

If I want to allow more users to see metadata about helm deployments via `helm ls -A` I need to grant them access to every secret in the entire cluster?

Would it not be possible to store metadata about deploys into a specific "helm-deploys" namespace and grant access to THAT instead? 

Am I missing something? I understand there may be sensitive info in the helm deploys and that is the reasoning to keep it all secret but this makes no sense to me.  Allowing certain people to possibly see leaked secrets vs. explicitly letting them see everything is worlds apart.